Gemfile

```
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 gem 'jekyll', '>= 3.9.5'
```

```suggestion
gem 'jekyll', '>= 3.10.0'
```

Based on https://pages.github.com/versions/